README.md

```diff
@@ -1,2 +1,6 @@
 # build-tools
-Various build tools
+Various build tools used as part of the M3DB project
+
+## Contents
+
+1. maptime - gometalinter plugin that detects Golang maps which have an instance of time.Time as part of the key
```

Comment:
could you add a sample gometalinter config to use this linter, e.g. of what I mean [here](https://github.com/m3db/m3db/blob/master/.metalinter.json#L4). Here's the upstream struct for configuring a [linter](https://github.com/alecthomas/gometalinter/blob/master/linters.go#L14)

Reply:
deleting this, your readme already does this.